### PR TITLE
Prevent protobuf nil pointer dereference

### PR DESCRIPTION
**Pull request by AMarcedone**

### CODE REVIEW

**Comment by AMarcedone** [comment]
Updated email address in the commits.

File: AUTHORS [hunk 1]
```diff
@@ -6,5 +6,6 @@
 # Name or Organization <email address>
 # The email address is not required for organizations.
 
+Antonio Marcedone <a.marcedone@gmail.com>
```
Comment (gdbelvin) [inline AUTHORS]: This should be Snap or Snap Inc.  
Also, please alphabetize

File: CONTRIBUTORS [hunk 2]
```diff
@@ -9,6 +9,7 @@
 # Names should be added to this file as:
 #     Name <email address>
 
+Antonio Marcedone <a.marcedone@gmail.com>
```
Comment (gdbelvin) [inline CONTRIBUTORS]: Please alphabetize

**Comment by gdbelvin** [comment]
Bump?

**Comment by AMarcedone** [comment]
Still waiting on this CLA, sorry!

**Comment by gdbelvin** [comment]
Let me know if there's anything we can do on our side to help.

**Comment by AMarcedone** [comment]
Update: I am still trying to get this CLA issues resolved asap.
Sorry for the delay.

**Comment by gdbelvin** [comment]
Thanks Antonio we look forward to working together!

**Comment by gdbelvin** [comment]
Happy to merge now if you update :-)

**Comment by AMarcedone** [comment]
Updated :)

**Pull request merged**